modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -266,7 +266,8 @@ export default {
     async edit(pieceId) {
       const doc = await apos.modal.execute(this.options.components.insertModal, {
         moduleName: this.moduleName,
-        docId: pieceId
+        docId: pieceId,
+        filterValues: this.filterValues
```

Comment:
I inadvertently broke editing the trash when I implemented apos.modal.execute by failing to pass this prop.